### PR TITLE
Scoping like cd 

### PR DESCRIPTION
**Pull request by oakeyc**

### CODE REVIEW

File: azclishell/configuration.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
     'example' : '::',
     'exit_code' : '$',
     'scope' : '%%',
-    'unscope' : '^^'
+    'unscope' : '..'
```
Comment (tjprescott) [inline azclishell/configuration.py:20]: So is it `%% ..` to go back or just `..`?

**Review by tjprescott — COMMENTED** [review]
Can you provide some examples of how the feature now works after this change?

File: azclishell/configuration.py (hunk 1 above)
Comment (oakeyc) [reply azclishell/configuration.py:20]: %% .. would go back
e.g. 
az>> vm .. vm
would result in 
az vm>>

Comment (tjprescott) [reply azclishell/configuration.py:20]: I'm... a little confused by that example.  So:
```
az>> %% vm
az vm>> %% ..
az>>
```
Yes?

Comment (oakeyc) [reply azclishell/configuration.py:20]: yes

**Review by tjprescott — APPROVED** [review]

**Pull request merged**